internal/provider/vaultsecrets/resource_vault_secrets_secret_test.go

```diff
@@ -13,6 +14,7 @@ import (
 
 func TestAccVaultSecretsResourceSecret(t *testing.T) {
 	testAppName := generateRandomSlug()
+	projectID := os.Getenv("HCP_PROJECT_ID")
```

Comment:
same concerns as above. I wonder if we should ensure project_id is not what we expected and not null vs adding the env as this can be impacted locally. The other test could be expect this project id  - might have to create it using the tf. thoughts?